### ISSUE TITLE
Uncaught TypeError: player.spriteThumbnails is not a function

### ISSUE THREAD

**Issue by venomone** [videojs/video.js#7786]
### Description

Hello,

I'm trying to integrate a plugin to my VideoJS setup to display a thumbnail preview if you hover over the scroll bar, therefor I'm using the following: https://github.com/phloxic/videojs-sprite-thumbnails

Currently, my player simply does not load at all, if I take a look at the Developer console of my browser I see:

    **Uncaught TypeError: player.spriteThumbnails is not a function**

Why that? Is this an issue that most likely comes from the plugin or from my setup?

This is how my VideoJS Setup currently look like:

```
         <video-js autoplay id=player class="vjs-fluid" controls="true">
                    <script src="{% static 'js/jquery-3.6.0.min.js' %}"></script>
                    <script src="{% static 'videojs/video.js' %}"></script>
                    <script src="{% static 'videojs/videojs-sprite-thumbnails.js' %}"></script>
                    <script src="{% static 'videojs/videojs-http-streaming.js' %}"></script>
                    <script>
                        videojs.Vhs.xhr.beforeRequest = function (options) {
                            options.headers = options.headers || {};
                            options.headers.Authorization = "{{ access_token }}";
                        };
                        var player = videojs('player', function () {
                        }, {
                            fill: true,
                            autoplay: true,
                            responsive: true,
                            fluid: true,
                            preferFullWindow: true,
                            preload: "auto",
                            enableLowInitialPlaylist: true,
                            html5: {
                                nativeControlsForTouch: false,
                                nativeAudioTracks: false,
                                nativeVideoTracks: false,
                                vhs: {
                                    enableLowInitialPlaylist: true,
                                    limitRenditionByPlayerDimensions: true,
                                    smoothQualityChange: true,
                                    overrideNative: false,
                                    handleManifestRedirects: true,
                                    useNetworkInformationApi: true,
                                }},
                            plugins: {
                                // default thumbnail settings for this player
                                spriteThumbnails: {
                                    width: 192,
                                    height: 108
                                }
                            },
                            function() {
                                this.on('error', function (event) {
                                    const time = this.currentTime();
                                    if (this.error().code === 3) {
                                        this.error(null).pause().load().currentTime(time).play();
                                    }
                                    if (this.error().code === 2) {
                                        this.error(null).pause().load().currentTime(time).play();
                                    }
                                    if (this.error().code === 1) {
                                        this.error(null).pause().load().currentTime(time).play();
                                    }
                                });
                            }
                        });
                        player.spriteThumbnails({
                            url: '{{ sprites_url }}',
                            width: 192,
                            height: 108,
                            interval: 30,
                        }).log.level('debug');
                        player.src({
                            src: '{{ playlist_url }}',
                            type: 'application/vnd.apple.mpegurl m3u8',
                            Authorization: true
                        });
                        player.play();
                    </script>
                </video-js>
```

### Steps to reproduce

Basically, it should be possible to use my code representation from above.
My versions:

VideoJs Version is 7.19.2
http-streaming version is: 2.14.2

### Errors

Uncaught TypeError: player.spriteThumbnails is not a function

### What version of Video.js are you using?

7.19.2

### Video.js plugins used.

http-streaming, spriteThumbnails

### What browser(s) including version(s) does this occur with?

Firefox 100

### What OS(es) and version(s) does this occur with?

Windows 10 & Ubuntu

**Comment by venomone** [comment]
Solved, Issue was on my side
